app/controllers/exercises_controller.rb

```diff
@@ -53,4 +53,22 @@ def show
       end
     end
   end
+
+  def set_disabled_statuses
+    @course = Course.find(params[:course_id])
+    @organization = @course.organization
+    authorize! :teach, @organization
+
+    action = params[:commit] == 'Disable selected' ? :disabled! : :enabled!
+    exercise_params = params[:course][:exercises]
+    exercise_params.reject!(&:blank?)
+
+    exercises = Exercise.where(id: exercise_params)
+    exercises.each do |ex|
+      ex.send(action)
+    end
```

Comment:
not too happy with this each loop. can't we just batch update `Exercise.where(id: exercise_params).update_all(disabled_status: 1)` (or by using the enums)